gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/TimeAwareRecursiveCopyableDataset.java

```diff
@@ -86,7 +86,14 @@ public void remove() {
   protected List<FileStatus> getFilesAtPath(FileSystem fs, Path path, PathFilter fileFilter) throws IOException {
     DateTimeFormatter formatter = DateTimeFormatter.ofPattern(datePattern);
     LocalDateTime endDate = LocalDateTime.now();
-    LocalDateTime startDate = endDate.minusDays(lookbackDays);
+    LocalDateTime startDate;
+    if (this.lookbackTime.endsWith("d")) {
+      startDate = endDate.minusDays(Long.parseLong(lookbackTime.substring(0, lookbackTime.length() - 1)));
```

Comment:
What about using Period like in SelectBetweenTimeBasedPolicy? I think that allows mixing like 7d6h.

Reply:
Good suggestion! Thanks!